ports/espressif/boards/waveshare_esp32s2_pico/sdkconfig

```diff
@@ -1,9 +1,11 @@
+#
+# SPI RAM config
+#
 CONFIG_ESP32S2_SPIRAM_SUPPORT=y
-
 CONFIG_DEFAULT_PSRAM_CLK_IO=30
 CONFIG_DEFAULT_PSRAM_CS_IO=26
-
 CONFIG_SPIRAM_TYPE_ESPPSRAM64=y
+CONFIG_SPIRAM_SIZE=8388608
```

Comment:
This page says it is 4MB: https://www.waveshare.com/wiki/ESP32-S2-Pico. But yours is 8MB? Is it clearly a genuine WaveShare board?

Reply:
3rd bulletpoint.

Reply:
8mb psram, 4mb flash.

Reply:
Ah, got it, sorry, thanks.